tests/common.py

```diff
@@ -405,6 +405,10 @@ def setUp(self):
     super().setUp()
     self.settings_mods = {}
     self.emcc_args = ['-Werror']
+    # We want to be strict about closure warnings in our test code.
+    # TODO(sbc): Remove this if we make it the default for `-Werror`:
+    # https://github.com/emscripten-core/emscripten/issues/16205):
+    self.ldflags = ['-sCLOSURE_WARNINGS=error']
```

Comment:
Isn't this the same as `IGNORE_CLOSURE_COMPILER_ERRORS` which is default to off? I actually don't remember when we added `CLOSURE_WARNINGS`. Maybe I'm not understanding how the two differ?

Reply:
`IGNORE_CLOSURE_COMPILER_ERRORS` I believe controls how closer handles what it considers errors, whereas  `CLOSURE_WARNINGS` control what it only considers to be warnings. 

I also think `IGNORE_CLOSURE_COMPILER_ERRORS` pre-dates `CLOSURE_WARNINGS` (and I'm not quite sure what happens if you ignore errors, or when you might want to).

In any case this change is about warnings.. errors are already fatal by default.